docker-compose.yml

```diff
@@ -71,7 +67,7 @@ services:
     volumes:
       - ckan_storage:/var/lib/ckan
       - ../:/usr/lib/adx
-      - ../ckan:/usr/lib/ckan/venv/src/ckan
+      - ./:/usr/lib/ckan
```

Comment:
I don't understand this change (as simillar one from line :44). Could you explain why we link the whole adr project directory as "ckan" to the container?

Reply:
rename to `/usr/lib/adx_develop`